### ISSUE TITLE
Gem should be able to respect app config file for couchdb connection

### ISSUE THREAD

**Issue by brlanier** [amco/couchdb_reindexer#1]
Would be nice if you didn't have to pass all the information that is already available in the couchdb.yml file on the command line. Its a security issue as the credentials remain in the shell history unless steps are taken to remove/minimize exposure. 
Plus it would be convenient since the info is already in the file.

**Comment by waxave** [comment]
Actually, you can use it without CLI

```ruby
require 'couchdb_reindexer'

OPTIONS = {
  database: "database",
  host: "127.0.0.1",
  username: "username",
  password: "password"
}

CouchdbReindexer::Indexer.new(OPTIONS).run
```
But maybe an option to run it with a config file is better.

**Comment by brlanier** [comment]
Sure, but why redefine all those values if you already have them defined in a config file somewhere. Just saying it would be nice to have the option to reuse something already in place. In the example above, you could load the config file into your options object, so that would probably work in that case. I know there is a new and improved syntax for this process in new rails, but here is an example from one of our apps modified to fit the scenario
```
OPTIONS ||= YAML.load(ERB.new(File.read(Rails.root.join('config', 'couchdb.yml'))).result)[Rails.env]
```
I just think the gem should try to reuse a defined file if it exists or have the option to.

**Comment by waxave** [comment]
Yes, well I actually wrote it based on another script, that's why I write as a CLI application, but I'll fix it
